portfolio/src/main/webapp/volleyball/volleyball.html

```diff
@@ -0,0 +1,959 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Volleyball</title>
+    <link rel="stylesheet" href="volleyball.css" />
+    <link rel="stylesheet" href="/../buttons.css" />
+  </head>
+  <body>
+    <div id="content">
+      <div>
+        <a href="/../index.html" class="button button-black">Home</a>
+      </div>
+      <h1>Here are some highlights of mine!</h1>
+      <!-- [html-validate-disable no-inline-style] -->
+      <!-- [html-validate-disable svg-focusable] -->
+      <!-- [html-validate-disable element-required-attributes] -->
+      <!-- [html-validate-disable no-deprecated-attr] -->
+      <div align="center">
+        <blockquote
+          class="instagram-media"
+          data-instgrm-permalink="https://www.instagram.com/p/B9mfcGXJTlJ/?utm_source=ig_embed&amp;utm_campaign=loading"
+          data-instgrm-version="12"
+          style="
```

Comment:
Can you put all styling into css file? That will help clean up the code a lot!

Reply:
Brett and I talked about this and since this is code copied from Instagram we can just ignore the check? But to make the code cleaner, I would suggest that you put the copied code in a separate `instagram.html` file, and then insert it into a `<div id="instagram">` in this `volleyball.html`. For example https://stackoverflow.com/questions/8988855/include-another-html-file-in-a-html-file?page=1&tab=votes#tab-top (jquery makes it a bit easier, but vanilla JS works fine too)